src/app/modules/form/public-form/public-form.service.ts

```diff
@@ -73,9 +73,11 @@ export const getCookieNameByAuthType = (
     | FormAuthType.SP
     | FormAuthType.CP
     | FormAuthType.MyInfo
-    | FormAuthType.SGID,
+    | FormAuthType.SGID
+    | FormAuthType.SGID_MyInfo,
 ): string => {
   switch (authType) {
+    case FormAuthType.SGID_MyInfo:
```

Comment:
should this now be `SGID_MYINFO_COOKIE_NAME`

Reply:
This is the cookie that determines whether the user is logged in/out. In this case, it's correct to clear the MyInfo login cookie as that contains the login credentials.

Reply:
Thanks! This should now be `SGID_MYINFO_LOGIN_COOKIE_NAME` correct